### PR TITLE
#9655 Added missing docker images

### PR DESCRIPTION
**Pull request by kanekoh**
#9655

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@jianlinliu, will you PTAL? Is this valid for 3.9 and later?

**Comment by jianlinliu** [comment]
@ganhuang PTAL.

File: install_config/install/disconnected_install.adoc [hunk 1]
```diff
@@ -207,6 +207,8 @@ $ docker pull registry.access.redhat.com/openshift3/ose-haproxy-router:<tag>
 $ docker pull registry.access.redhat.com/openshift3/ose-keepalived-ipfailover:<tag>
 $ docker pull registry.access.redhat.com/openshift3/ose-pod:<tag>
 $ docker pull registry.access.redhat.com/openshift3/ose-sti-builder:<tag>
+$ docker pull registry.access.redhat.com/openshift3/ose-template-service-broker:<tag>
+$ docker pull registry.access.redhat.com/openshift3/ose-web-console:<tag>
```
Comment (yapei) [inline install_config/install/disconnected_install.adoc]: web console image is correct

Comment (wzheng1) [reply install_config/install/disconnected_install.adoc]: ose-template-service-broker is correct.

**Comment by stuartchuan** [comment]
@anpingli please help review oauth-proxy image, thx.

**Comment by anpingli** [comment]
LGTM

**Comment by kalexand-rh** [comment]
Thank you all!

Will one of you please confirm that this change is valid in 3.9 and 3.10? Thanks. :)

**Comment by yapei** [comment]
I could confirm these are valid for both 3.9 and 3.10

**Comment by anpingli** [comment]
@kalexand-rh  There may be some new images in v3.10.  For example: metrics-schema-installer.

**Comment by kalexand-rh** [comment]
@anpingli, I'll merge it as-is to 3.9 and 3.10. Do you have a suggestion for who can help us get an updated list of images before release?

**Pull request merged**